docs/reference/ingest/processors/inference.asciidoc

```diff
@@ -0,0 +1,107 @@
+[role="xpack"]
+[testenv="basic"]
+[[inference-processor]]
+=== {infer-cap} Processor
+
+Uses a pre-trained {dfanalytics} model to infer against the data that is being 
+ingested in the pipeline.
+
+
+[[inference-options]]
+.{infer-cap} Options
+[options="header"]
+|======
+| Name               | Required  | Default                        | Description
+| `model_id   `      | yes       | -                              | (String) The ID of the model to load and infer against.
```

Comment:
This formatting isn't working in the preview.  Not sure if this will fix the problem?:
```suggestion
| `model_id`      | yes       | -                              | (String) The ID of the model to load and infer against.
```